cli/src/template.rs

```diff
@@ -91,13 +91,20 @@ impl CircuitTemplate {
         paths.push(DEFAULT_TEMPLATE_DIR.to_string());
         let manager = CircuitTemplateManager::new(&paths);
         let possible_values = manager.list_available_templates()?;
-        if !possible_values.iter().any(|(stem, _)| stem == name) {
-            return Err(CliError::ActionError(format!(
-                "Template with name {} was not found. Available templates: {:?}",
-                name, possible_values
-            )));
-        }
-        let template = manager.load(name)?;
+        let template = manager.load(name).map_err(|err| {
+            let named_path = Path::new(name);
+            if named_path.has_root() {
```

Comment:
Template load actually supports both passing in with the root or without https://github.com/Cargill/splinter/blob/main/libsplinter/src/circuit/template/mod.rs#L183 , so instead this should check the following https://github.com/Cargill/splinter/blob/main/libsplinter/src/circuit/template/mod.rs#L180

Reply:
Fixed